### ISSUE TITLE
Atom is not shown anywhere on screen

### ISSUE THREAD

**Issue by InfInt** [atom/atom#8737]
I can start a atom process either by console or desktop shortcut, but the whole UI won't show up. I can see the the window preview in the taskbar though. 
I am using:
* Windows 10 build 10532
* Atom 1.0.11

A reinstallation didn't work, too.

**Comment by Zireael07** [comment]
What is your GPU? Is it an integrated one?

**Comment by InfInt** [comment]
I have a nVidia GT 130.

**Comment by Zireael07** [comment]
There is an issue related to nVidia drivers, but it's about crashing. Interesting.

**Comment by KENJU** [comment]
I'm not sure about windows, but in Mac, you can run Console.app and see the log. When I had bug, I also could not find out because either console or desktop shortcut show nothing, but I could see the log from Console.app. If you have similar softwares on your WIndows I think you can find out what is going on behind it...

**Comment by KENJU** [comment]
Are you opening in developer mode?

**Comment by InfInt** [comment]
I've started atom via windows command line, but it doesn't show any errors there and no window is visible. In the task manager I can see a running process, that's all. My machine is running in developer mode, but not atom itself.

The curious thing is that atom was working fine on my machine before. I have not change any global configuration or software. It just stopped showing a gui, suddenly.

**Comment by mnquintana** [comment]
Can you try quitting Atom, and starting it in safe mode? (`atom --safe`)

Do you have a multi-monitor setup? Are you running Windows in a VM?

**Comment by InfInt** [comment]
Safe mode doesn't work either. Yes, I've two monitors each 1920x1200. No, Windows 10 is running as host on an iMac 24" early 2009.

**Comment by yoannfleurydev** [comment]
I have the same problem. When my laptop is plugged into my second screen to do dual screen, no problem, I can open Atom and see it on my laptop screen (which is not my primary screen but is on the right). The problem is when I unplug my laptop. Atom is open but I have only the preview in taskbar. When I click on the preview, the Atom's window goes on the right, then I click again and the window is minimized.

I'm running Windows 8.1 with a NVIDIA 840M GPU.

Hope this details can help, and hope to find a solution soon, because I can't use Atom when my PC is not plugged into the other screen.

**Comment by InfInt** [comment]
I have tried it with one screen attached. No change unfortunately.

**Comment by KENJU** [comment]
> The curious thing is that atom was working fine on my machine before. I have not change any global configuration or software. It just stopped showing a gui, suddenly.


Have you cloned any repository, like atom?

( If you have cloned any third-party plugin or repository, how about checking the package.json, and see you have all dependencies in your machine (globally) or in the repository (locally). If you don't, run `npm install package-name` to install. )

That's all what I come up with.

**Comment by InfInt** [comment]
I just found a solution to my problem. The atom window was rendered outside any screen coordinates. I did the following steps to solve it:

1. Alt + Tab to choose the atom window
2. Alt + Space to open the context menu
3. Press 'm' to select move
4. Press any arrow key once
5. Move your mouse (_The misplaced window will follow your cursor._)
6. Open a new file or project and close it again to save the new window state.

**Comment by yoannfleurydev** [comment]
I just find a solution too :

Just **Right Click** on the **Windows taskbar**, then open the **Windows Task Manager**. 
Next step is to select one subprocess of Atom and **Right Click** on it, and then select **Maximize**.
The Atom's window is now on the screen.

**Comment by vgf89** [comment]
Alternatively, hover over the icon in your taskbar, right click the window preview, and click Maximize there.

**Comment by corleyma** [comment]
Just noting that I had the exact same thing happen running Ubuntu 14.04.  Not sure how it happened, but one second I'm using Atom, the next the window is gone but the app is still open.  Fullscreen mode worked, but no window was visible outside fullscreen.  The solution proposed by @InfInt worked for me: intriguingly, the window was being rendered but only as a very tiny line, that I had to drag to expand (presumably maximizing would also work).

**Comment by tonywmnix** [comment]
This happens to me frequently when I open Atom while docked and displayed on my second monitor (not marked as primary) and when I undock and use my laptop screen the next time i start Atom it's off screen and I use the procedures mentioned above to move the window. Seems like electron might be the culprit.

**Comment by lorencole** [comment]
I also had this problem on Ubuntu 14.04 LTS, but instead of being draw off screen, the window was sized to something like 0 x 60 px so only a short thin line with a drop shadow displayed. Clicking on this I was able to resize the window. 

The problem started when I did a find and replace of " to \".

**Comment by amertak** [comment]
I had this issue also, thanks for the solutions.
I used 34' wide monitor yesterday and I think it remembered its position.

**Comment by lee-dohm** [comment]
This is a duplicate of #6939. You may want to subscribe there for updates.

**Comment by vijay-stayntouch** [comment]
@InfInt thanks a lot mate!!

**Comment by audstanley** [comment]
Had the same issue: Dual Display (Apple 30" on PC), running on Nvidia.  I started task manager, down arrow on Atom, right click the Atom window, then maximize, drag the maximized window out of maximize, now when I re-open Atom, all is fine.

**Comment by archerzhaocs** [comment]
Thanks a lot!  You saved me! Cost me at least 2 hours to try to find out what happens. I reinstalled atom with multiple version(including beta version) many times. Delete ".atom" folder many times. But nothing helps.

I have dual screen. The OS is mac os sierra.

**Comment by mrejster** [comment]
In my case, moving the window helped revealing my actual problem: The window was shrunk to a thin vertical line. Expanding the line brought Atom back!

**Comment by sagar74132** [comment]
After installation just right click on the taskbar icon of Atom and select "**Pin to taskbar**" . That's it . It is working in my case .
tested in Win 10

**Comment by brownwrap** [comment]
I don't know how the issues work. I am on a Linux box, but with the same problem. Should I open a new ticket? I am not at all familiar with Atom. Two users requested it. The RPM didn't work, so I downloaded the source. I installed Node and Yargs, then built Atom. I start Atom, 'ps -ef' says it is running, but I don't get any kind of window. Thanks.

**Comment by varaprasadreddy9676** [comment]
i have the same problem for months after trying disperately
i found the solution not exact but works very fine
just right click on your atom application and select run with integrated graphics processor.

**Comment by NaimBendjebbour** [comment]
I have the same problem to solve it, go to userFolder/Appdata/Roaming/Atom/ Delete all file in this folder.
When you restart atom all work fine !

**Comment by elmatou** [comment]
with a fresh Ubuntu install, with both the atom snap and deb package (1.23.3) , I ran in the same issue.
the windows was something with 0px wide... "Super + up arrow" did the trick to maximize the window

**Comment by varaprasadreddy9676** [comment]
yaay!!! it worked for me thank you very much @NaimBendjebbour

**Comment by goalaleo** [comment]
For OS X users: For me the Atom window was also shrunk into a tiny bad that I couldn't find.
Alt + Tab to select atom, then from top Window -> Zoom to make the Atom window big again

**Comment by coldfire15** [comment]
I tried windows+left arrow with an application opened (browser), windows proposes to choose a new application for the right side and you choose atom and you maximize

**Comment by mindfulloffire** [comment]
This was happening to me as well.  With dexpot I was able to maximize the window and atom would appear, but if closed from max it would still be "missing" from the screen(s). To get it to appear it needs to be "grabbed" from the max position by clicking on the top bar and moved anywhere onto the visible screen before closing.  The next atom window opened in a visible position.

**Comment by padmarajnidagundi** [comment]
Imagine: It happens when you're using external multiple screens to the PC/Laptop and the next day you will not use external multiple screens and start PC/Laptop `and you will notice "Atom is not shown anywhere on screen"

The solution is to reactivate or bring back your Atom window to your screen.

- Alt + Tab to choose the atom window
- Alt + Space to open the context menu
- Press 'm' to select move
- Press any arrow key once
- Move your mouse (The misplaced window will follow your cursor.)
- Open a new file or project and close it again to save the new window state.
- Now go to File menu and use the Open folder or Reopen project to reach recent projects on you are working

**Comment by Kevin-RS** [comment]
> I just found a solution to my problem. The atom window was rendered outside any screen coordinates. I did the following steps to solve it:
> 
> 1. Alt + Tab to choose the atom window
> 2. Alt + Space to open the context menu
> 3. Press 'm' to select move
> 4. Press any arrow key once
> 5. Move your mouse (_The misplaced window will follow your cursor._)
> 6. Open a new file or project and close it again to save the new window state.

Thanks!!

**Comment by mdesantis** [comment]
> with a fresh Ubuntu install, with both the atom snap and deb package (1.23.3) , I ran in the same issue.
> the windows was something with 0px wide... "Super + up arrow" did the trick to maximize the window

@elmatou you saved my day

**Comment by toddhd** [comment]
This just happened to me on a Mac. I minimized the window and then boom - it was gone, even though the process was running. I rebooted and no change. 
Turns out that it had somehow become a thin line instead of a window, maybe one pixel wide and a few pixels high, and was on the bottom left edge of my screen. Mousing over the tiny line allow me to grab it, then I could resize it normally. I do have two monitors, this was on the screen that was actually part of my laptop, not the external monitor.

**Comment by arunmummidi** [comment]
I faced same issue. But I was able to fix it by following below steps:

    Go to View > Show all tabs

This will pop up all the tabs. Then you can use Atom normally.

Thanks!
Arun

**Comment by sd031** [comment]
Just happened with me and surprisingly I can see top options by but not the main UI not showing anywhere. Totally clueless. Restarting and reinstalling doesn't work.

**Comment by davidharman** [comment]
> This just happened to me on a Mac. I minimized the window and then boom - it was gone, even though the process was running. I rebooted and no change.
> Turns out that it had somehow become a thin line instead of a window, maybe one pixel wide and a few pixels high, and was on the bottom left edge of my screen. Mousing over the tiny line allow me to grab it, then I could resize it normally. I do have two monitors, this was on the screen that was actually part of my laptop, not the external monitor.


----------
Thanks, this worked for me, I also had the same thin line in the bottom of my primary screen, found it, resized, all back again....

**Comment by YenHub** [comment]
Wow, 4 years down the line and this is still a thing! 

@InfInt Kudos++

This doesn't appear to be a windows thing, since we have the Ubuntu & Mac crowd picketing too..

Anyone got any ideas where to start looking at this? It looks to be synonymous with disconnecting from an external monitor.. ?

**Comment by ShayGaming24** [comment]
> I just found a solution to my problem. The atom window was rendered outside any screen coordinates. I did the following steps to solve it:
> 
> 1. Alt + Tab to choose the atom window
> 2. Alt + Space to open the context menu
> 3. Press 'm' to select move
> 4. Press any arrow key once
> 5. Move your mouse (_The misplaced window will follow your cursor._)
> 6. Open a new file or project and close it again to save the new window state.

Thanks so much - this worked for me!

**Comment by YenHub** [comment]
Turns out this is a chromium issue - nothing to do with Atom at all!! 

Solved! (Kind of..?)

> I just found a solution to my problem. The atom window was rendered outside any screen coordinates. I did the following steps to solve it:  
>
> Alt + Tab to choose the atom window  
Alt + Space to open the context menu  
Press 'm' to select move  
Press any arrow key once  
Move your mouse (The misplaced window will follow your cursor.)  
Open a new file or project and close it again to save the new window state.

**Comment by christophcemper** [comment]
I have this problem too, for months.

In my case the "Move" is not available, so I can either have the window minimized, or maximized, no size between

**Comment by emcgal** [comment]
> I have the same problem to solve it, go to userFolder/Appdata/Roaming/Atom/ Delete all file in this folder.
> When you restart atom all work fine !

I was having the same issue, using 2 screens (extended) and my atom decided to stop showing. Re-installing didn't change anything nor disconnecting my big screen and just using the laptop. Was open as such and tried some of the other suggestions but this one from @NaimBendjebbour Worked a treat. I was on the verge of switching to another editor. Thanks a mil :-)

**Comment by jayzhou215** [comment]
> I also had this problem on Ubuntu 14.04 LTS, but instead of being draw off screen, the window was sized to something like 0 x 60 px so only a short thin line with a drop shadow displayed. Clicking on this I was able to resize the window.
> 
> The problem started when I did a find and replace of " to ".

I had this problem on mac 10.14.6, atom version 1.42.0,  also window was sized to something like 0*60 px. Choose the thin line, and select window zoom, and the window show as normal.

**Comment by VictorImaginanet** [comment]
> I just found a solution to my problem. The atom window was rendered outside any screen coordinates. I did the following steps to solve it:
> 
> 1. Alt + Tab to choose the atom window
> 2. Alt + Space to open the context menu
> 3. Press 'm' to select move
> 4. Press any arrow key once
> 5. Move your mouse (_The misplaced window will follow your cursor._)
> 6. Open a new file or project and close it again to save the new window state.

OMG thank you so much, it worked for me

**Comment by linberg** [comment]
MacOS 10.15.3 and Atom 1.43.0 and having the same issue. 

To be sure, to ensure this doesn't happen when disconnecting from the external monitor, you either have to quit Atom completely before disconnecting, or move all open windows to the laptop screen?

**Comment by abir286** [comment]
same problem with me!!

**Comment by David-OpenOne** [comment]
> I just found a solution to my problem. The atom window was rendered outside any screen coordinates. I did the following steps to solve it:
> 
> 1. Alt + Tab to choose the atom window
> 2. Alt + Space to open the context menu
> 3. Press 'm' to select move
> 4. Press any arrow key once
> 5. Move your mouse (_The misplaced window will follow your cursor._)
> 6. Open a new file or project and close it again to save the new window state.

I had the same issue, and trying the installs / re-installs / purges / chrome install etc did not work.

Then tried this solution by **InfInt** and worked. Such a simple solution but has not been fixed - using UBUNTU 18.04 LTS, dual screens.

Thank you, thank you, thank you - I see that there were others with the same solution.

**Comment by nturumel** [comment]
Please try this for windows .. 
1. alt + tab  to select the atom window
2. alt + space ... this should open a menu for resizing the window  ..
3. select maximise

Good to Go!!!
😊

**Comment by fitzzz** [comment]
> Can you try quitting Atom, and starting it in safe mode? (`atom --safe`)
> 
> Do you have a multi-monitor setup? Are you running Windows in a VM?

This helped me get in and realize it was a buggy theme that caused my issue. Then I went to the atom config and removed the themes. Much appreciated

**Comment by denso666** [comment]
Once atom is open, right click on the icon on your taskbar and open a new window. This solved the problem.
Kde Neon 5.21.

**Comment by flukymaze** [comment]
> Alternatively, hover over the icon in your taskbar, right click the window preview, and click Maximize there.

This is the easiest and most efficient way to solve the problem.

**Comment by ShaikMasthanshareef** [comment]
when i enter my code in atom app it is not show in output in broswer an html data base

**Comment by ShaikMasthanshareef** [comment]
when i enter my code in atom app it is not show in output in browser an html data base

**Comment by ShaikMasthanshareef** [comment]
![Screenshot (31)](https://user-images.githubusercontent.com/89543273/132116986-360d8216-a84c-4469-8af8-8b0f47a0e8fe.png)

> when i enter my code in atom app it is not show in output in broswer?

**Comment by icecream17** [comment]
@ShaikMasthanshareef 
If you get a virus protection like MacAfee but don't activate it, you have no protection

So if you can't activate it right now, it's better to enable the default Windows Security

**Comment by cevherdogan** [comment]
mine was a space issue. i had the Atom app up and running and won't be able to see the window. It might have been a dot on the left screen and perhaps somewhere something got stuck in its previous state telling it to open there. I am assuming this form my readings from earlier folks. but other things might have been in play from the start as I did not have Admin password on this Mac and had to install Atom in another folder rather than /Applications. i constantly had space issues and noticed some apps rollback might not have happened cleanly. Even package uninstallation was not working right it was coming back repeatedly after restarting Atom. Long story short, several reinstalls later I still did not have it working. I searched and saw Android Studio which I happened to have used couple years ago taking the largest space. Cleaned up and reinstalled Atom, and worked fine :) Thank you for all teh above shares, that helped in great deal!!
